script/prerender/fileUtils.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs';
+import flow from 'lodash/fp/flow';
+import path from 'path';
+
+const ASSET_DIR = path.resolve(__dirname, '../../build');
+
+const prefixPath = (filePath: string) => path.join(ASSET_DIR, filePath);
+const directoryExists = (filePath: string) => fs.existsSync(filePath) && fs.lstatSync(filePath).isDirectory();
+
+export const assetDirectoryExists = flow(prefixPath, directoryExists);
+
+function makeDirectories(filepath: string) {
+  const dirname = path.dirname(filepath);
+  if (fs.existsSync(dirname)) {
+    return true;
+  }
```

Comment:
Might check if the directory can be written to using `fs.access`

Reply:
assuming fs.mkdirSync throws when it isn't writable, i'm fine with that, it should never happen.